### PR TITLE
Integrate Equipment API with Swagger

### PR DESCRIPTION
**Pull request by glauco**
This commit integrates Swagger with Equipment API.

It also structures the project in models, controllers and validators, 
each one with a single responsibility.

### CODE REVIEW

**Comment by andressa** [comment]
:+1:

**Comment by jeffhsta** [comment]
:+1:

**Comment by waldemarnt** [comment]
:+1:

File: validators/equipment.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import Joi from 'joi';
+
+const EquipmentValidator = () => {};
```
Comment (bltavares) [inline validators/equipment.js]: Is the plan to use classes here? Maybe on this case it would be better to use ES6 class syntax.

Comment (glauco) [inline validators/equipment.js]: We tried, but there were some conflicts between `babel` and our classes.

At the end, we decided to follow `hapi` standards.

Comment (bltavares) [inline validators/equipment.js]: Ok.

**Pull request merged**